connector/launchers/runtime-embedded/build.gradle.kts

```diff
@@ -17,6 +17,7 @@ dependencies {
         exclude(group = "org.eclipse.edc", "data-plane-selector-client")
     }
 
+    runtimeOnly(project(":core:data-migration"))
```

Comment:
just a nit: it's a database schema migration module, not a data migration one ;)